src/lib/components/Button/Button.jsx

```diff
@@ -25,7 +25,6 @@ export const Button = React.forwardRef((props, ref) => {
     priority,
     size,
     startCorner,
-    type,
```

Comment:
Here I removed `type` from `props`, but I kept it in `defaultProps` and `propTypes`. The reason is that the default value for `type` is different among browsers ("button" in IE and "submit" in other browsers) which can lead into different behaviour. Is it ok?